modules/system/defaults/NSGlobalDomain.nix

```diff
@@ -27,7 +87,35 @@ in {
       type = types.nullOr (types.enum [ "Dark" ]);
       default = null;
       description = ''
-        Set to 'Dark' to enable dark mode, or leave unset for normal mod.
+        Set to 'Dark' to enable dark mode, or leave unset for normal mode.
+      '';
+    };
+
+    system.defaults.NSGlobalDomain.AppleAccentColor = mkOption {
+      type = types.coercedTo
+        (types.nullOr (types.enum (builtins.attrNames appleSystemColors)))
+        coerceAppleAccentColor
+        (types.nullOr types.int);
+      default = null;
+      description = ''
+        Configures the color of native controls.
+        Catalina: The default is Blue
+        Big Sur: The default is multicolor
+      '';
+    };
+
+    system.defaults.NSGlobalDomain.AppleHighlightColor = mkOption {
+      type = types.coercedTo
+        (types.nullOr rgbDecimal)
```

Comment:
This should probably be updated to be able to accept either one of the named color strings or the rgb attrset for convenience